# Genotype imputation

Chip genotype data processing and QC Samples were genotyped with Illumina \(Illumina Inc., San Diego, CA, USA\) and Affymetrix arrays \(Thermo Fisher Scientific, Santa Clara, CA, USA\). Genotype calls were made with GenCall and zCall algorithms for Illumina and AxiomGT1 algorithm for Affymetrix data. Chip genotyping data produced with previous chip platforms and reference genome builds were lifted over to build version 38 \(GRCh38/hg38\) following the protocol described here: dx.doi.org/10.17504/protocols.io.nqtddwn. In sample-wise quality control, individuals with ambiguous gender, high genotype missingness \(&gt;5%\), excess heterozygosity \(+-4SD\) and non-Finnish ancestry were excluded. In variant-wise quality control variants with high missingness \(&gt;2%\), low HWE P-value \(&lt;1e-6\) and minor allele count, MAC&lt;3 were excluded. Chip genotyped samples were pre-phased with Eagle 2.3.5 \([https://data.broadinstitute.org/alkesgroup/Eagle/](https://data.broadinstitute.org/alkesgroup/Eagle/)\) with the default parameters, except the number of conditioning haplotypes was set to 20,000.

Genotype imputation with the population-specific [SISu v3 reference panel ](sisu-reference-panel.md)High-coverage \(25-30x\) WGS data used to develop the SISu v3 reference panel \(PÃ¤rn et al., manuscript in preparation\) were generated at the Broad Institute of MIT and Harvard and at the McDonnell Genome Institute at Washington University; and jointly processed at the Broad Institute. Variant call set was produced with GATK HaplotypeCaller algorithm by following GATK best-practices for variant calling. Genotype-, sample- and variant-wise QC was applied in an iterative manner by using the Hail framework \([https://github.com/hail-is/hail](https://github.com/hail-is/hail)\) v0.1 and the resulting high-quality WGS data for 3,775 individuals were phased with Eagle 2.3.5 as described above. Genotype imputation was carried out by using the population-specific SISu v3 imputation reference panel with Beagle 4.1 \(version 08Jun17.d8b, [https://faculty.washington.edu/browning/beagle/b4\_1.html](https://faculty.washington.edu/browning/beagle/b4_1.html)\) as described in the following protocol: dx.doi.org/10.17504/protocols.io.nmndc5e. Post-imputation quality-control involved checking expected conformity of the imputation INFO-value distribution, MAF differences between the target dataset and the imputation reference panel and checking chromosomal continuity of the imputed genotype calls.

Optional: Post-imputation quality control also involved excluding variants imputed with imputation INFO&lt;0.7.

