# Quality control

This is a description of the quality control needed before running the GWAS. 

In summary, we removed xx samples who were either of non-Finnish ancestry or twins/duplicates. Finnish ancestry was assessed with a combination of [PCA](quality-checks.md) and a Bayesian method for outlier detection. Related or duplicate samples were detected using [kinship analysi](quality-checks.md)s. We also excluded xx samples that had missing minimum phenotype data.

## Sample QC and PCA

Our data set initially consists of 102,739 samples, of which we keep 100,355 after removing duplicates. 

After this step we need to exclude samples that do not have Finnish ancestry. After filtering for high quality HQ variants \(36, 073 variants\) we merge the data set with the thousand genomes data. At this point we perform a PCA on the merged data set and use a bayesian approach to determine outliers. This process allows us to identify samples from outside the Central/Northern European region \(1023\), however Western European and GB samples are still present, but are not enough to drive a signal in the PCA. Thus we use a different approach; we run a PCA on the 99333 samples left and we project the 98 FIN and 89 EUR samples from the thousand genomes project who survived round one onto the same space. Then, for each Finngen sample, we calculate its mahalanobis distance to the FIN and EUR centroid. The distance is mapped to a probability with a chi squared distribution with 3 degrees of freedom. Then, we define as being Finns, those sample for whom the relative probability of being Finnish vs European is &gt; 95%. This leaves us with 98644 samples.

The PCA for population structure has been run in the following way: Variant filtering and LD pruning The following filters were applied:

* Exclusion of chromosome X
* Exclusion of variants with info score &lt; 0.95
* Exclusion of variants with missingness &gt; 0.01 \(based on the GP,see conversion\)
* Exclusion of variants with MAF &lt; 0.05
* LD pruning with window 500kb, step 50kb, $$R^2 $$ filter of 0.1

  This filtering step produced 36,944 variants, that were used for the rest of the analysis

  PCA outlier detection

  Then, FinnGen data was merged with the 1k genome project \(1kgp\) data, using the variants mentioned above. A round of PCA was performed and a bayesian algorithm was used to spot outliers. This process got rid of 4,686 outliers, of which 2,292 are from the FinnGen samples. The figure below shows the scatter plots for the first 3 PCs. Outliers, in red, are separated from the FinnGen blue cluster.

While the method automatically detected as being outliers the 1kgp samples with non European and southern European ancestries, it did not manage to exclude some samples with Western European origins. Since the signal from these sample would have been too small to allow a second round to be performed without detecting substructures of the Finnish population, another approach was used. The FinnGen samples that survived the first round were used to compute another PCA. The EUR and FIN 1kgp samples were then projected onto the space generated by the first 3 PCs. Then, the centroid of each cluster was calculated and used it to calculate the squared mahalanobis distance of each FinnGen sample to each of the centroids. Being the squared distance a sum of squared variables \(with unitary variance, due to the mahalanobis distance\), we could see it as a sum of 3 independent squared variables. This allowed to map the squared distance into a probability \(chi squared with 3 degrees of freedom\). Therefore, for each cluster, a probability of being part of it was computed. Then, a threshold of 0.95 was used to exclude FinnGen samples whose relative chance of being part of the Finnish cluster was below the level. This method produced another 588 outliers. The figure below shows the first three principal components.

FIN 1kgp samples are in purple, while EUR 1kgp sample are in Blue. Samples in green are FinnGen samples who are flagged as being non Finnish, while red ones are considered Finnish.

## Kinship

Then all pairs of FinnGen samples up to second degree were returned. The figure below shows the distribution of kinship values. \Then, the previously defined “non Finnish” samples were excluded and 2 algorithms were used to return a unique subset of unrelated samples:

* one called ​greedy ​would continuously remove the highest degree node from the network of relations, until no more links are left in the network
* one called ​native ​, based on a native implementation of python’s ​networkx ​package, performed on each subgraph of the network.

  The largest independent set of either algorithm would be used to keep those samples, while flagging the others as “outliers” for the final PCA.

  Then, the subset of outliers who also belong to the set of duplicates/twins was identified.

## 

## Final PCA

To compute the final step the FinnGen samples were ultimately separated in three groups:

* 131,863 ​inliers​ . These are unrelated samples with Finnish ancestry
* 46,916 ​outliers ​who are of non duplicate samples with Finnish ancestries, but who are

  also related to the inliers

* 4,915 ​rejected ​samples, who are either of non Finnish ancestry or are twins/duplicates

  with relations to other samples

  Finally, the PCA for the inliers was calculated, and then outliers were projected on the same

  same, allowing to calculate covariates for a total of ​178,779​ samples.

## Sample filtering based on phenotype data

Of the 178,779 non-duplicate population inlier samples from PCA, we excluded 1,880 samples from analysis because of missing minimum phenotype data or a mismatch between imputed sex and sex in registry data. ​A total of 176,899 samples was used for core analysis.

## Further info 

### Bayesian outlier detection

Code for the method can be found here:​[ github.com/FINNGEN/pca\_outlier\_detection](https://github.com/FINNGEN/pca_outlier_detection). 

Official documentation from the original developers of the algorithm can be found here: [http://www.well.ox.ac.uk/~spencer/Aberrant/aberrant-manual.pdf](http://www.well.ox.ac.uk/~spencer/Aberrant/aberrant-manual.pdf). 

### Centroid based outlier detection

![Principal components 1-3, with FinnGen&apos;s Finnish individuals shown in red, FinnGen outliers in blue, and thousand genomes Finnish samples labelled in purple, Western European in green. ](../../.gitbook/assets/screenshot-2019-12-23-at-12.23.44.png)



This figure shows how the centroid based outlier detection works by plotting the distribution of the first 3 components of the PCA. Purple and green dots represent samples of Finnish and Western European respectively from the thousand genome data set. The blue dots are Finngen samples who have been found to be more likely to belong to the EUR group rather than to the Finnish one. Dots in red on the other hand are labelled as belonging to the Finnish centroid.

We can see that the Finngen samples labelled as EUR are extremely close to the EUR centroid in the first two components.



